mixitup/index.html

```diff
@@ -663,142 +77,29 @@ <h3 class="nameOnModal">Priyanka Oberoi</h3>
                     <div class="row">
                         <div class="col-lg-4 col-md-4 col-sm-4">
                             
-                            <img class="img-responsive img-centered img-circle" src="BioPics/Burton.jpg" alt="Burton Reist">
+                            <img class="img-responsive img-centered img-circle" src="BioPics/{{picture_file}}" alt="{{first_name}} {{last_name}}">
                             
                         </div>
                         <div class="col-lg-7  col-md-7 modalTextStyles" style="text-align:left; color: black">
 
                                 <!-- Bio Details Go Here -->
 
-                                <h3 class="nameOnModal">Burton Reist</h3>
-                                <h4>Director of External Affairs</h4>
+                                <h3 class="nameOnModal">{{first_name}} {{last_name}}</h3>
```

Comment:
It seems like since the modal is a relatively limited portion of content, using inheritance is a fine way to go about it, but I'll leave it up to you. Either way, can we stick with `classes-with-dashes` style rather than `camelCase`?